test/jdk/sun/net/www/http/HttpClient/KeepAliveTest.java

```diff
@@ -0,0 +1,1103 @@
+ * @run main/othervm KeepAliveTest 86
+ * @run main/othervm KeepAliveTest 87
+ * @run main/othervm KeepAliveTest 88
+ * @run main/othervm KeepAliveTest 89
+ * @run main/othervm KeepAliveTest 90
+ * @run main/othervm KeepAliveTest 91
+ * @run main/othervm KeepAliveTest 92
+ * @run main/othervm KeepAliveTest 93
+ * @run main/othervm KeepAliveTest 94
+ * @run main/othervm KeepAliveTest 95
+ * @run main/othervm KeepAliveTest 96
+ * @run main/othervm KeepAliveTest 97
+ * @run main/othervm KeepAliveTest 98
+ * @run main/othervm KeepAliveTest 99
+ * @run main/othervm KeepAliveTest 100
+ * @run main/othervm KeepAliveTest 101
+ * @run main/othervm KeepAliveTest 102
+ * @run main/othervm KeepAliveTest 103
+ * @run main/othervm KeepAliveTest 104
+ * @run main/othervm KeepAliveTest 105
+ * @run main/othervm KeepAliveTest 106
+ * @run main/othervm KeepAliveTest 107
+ * @run main/othervm KeepAliveTest 108
+ * @run main/othervm KeepAliveTest 109
+ * @run main/othervm KeepAliveTest 110
+ * @run main/othervm KeepAliveTest 111
+ * @run main/othervm KeepAliveTest 112
+ * @run main/othervm KeepAliveTest 113
+ * @run main/othervm KeepAliveTest 114
+ * @run main/othervm KeepAliveTest 115
+ * @run main/othervm KeepAliveTest 116
+ * @run main/othervm KeepAliveTest 117
+ * @run main/othervm KeepAliveTest 118
+ * @run main/othervm KeepAliveTest 119
+ * @run main/othervm KeepAliveTest 120
+ * @run main/othervm KeepAliveTest 121
+ * @run main/othervm KeepAliveTest 122
+ * @run main/othervm KeepAliveTest 123
+ * @run main/othervm KeepAliveTest 124
+ * @run main/othervm KeepAliveTest 125
+ * @run main/othervm KeepAliveTest 126
+ * @run main/othervm KeepAliveTest 127
+ * @run main/othervm KeepAliveTest 128
+ * @run main/othervm KeepAliveTest 129
+ * @run main/othervm KeepAliveTest 130
+ * @run main/othervm KeepAliveTest 131
+ * @run main/othervm KeepAliveTest 132
+ * @run main/othervm KeepAliveTest 133
+ * @run main/othervm KeepAliveTest 134
+ * @run main/othervm KeepAliveTest 135
+ * @run main/othervm KeepAliveTest 136
+ * @run main/othervm KeepAliveTest 137
+ * @run main/othervm KeepAliveTest 138
+ * @run main/othervm KeepAliveTest 139
+ * @run main/othervm KeepAliveTest 140
+ * @run main/othervm KeepAliveTest 141
+ * @run main/othervm KeepAliveTest 142
+ * @run main/othervm KeepAliveTest 143
+ * @run main/othervm KeepAliveTest 144
+ * @run main/othervm KeepAliveTest 145
+ * @run main/othervm KeepAliveTest 146
+ * @run main/othervm KeepAliveTest 147
+ * @run main/othervm KeepAliveTest 148
+ * @run main/othervm KeepAliveTest 149
+ * @run main/othervm KeepAliveTest 150
+ * @run main/othervm KeepAliveTest 151
+ * @run main/othervm KeepAliveTest 152
+ * @run main/othervm KeepAliveTest 153
+ * @run main/othervm KeepAliveTest 154
+ * @run main/othervm KeepAliveTest 155
+ * @run main/othervm KeepAliveTest 156
+ * @run main/othervm KeepAliveTest 157
+ * @run main/othervm KeepAliveTest 158
+ * @run main/othervm KeepAliveTest 159
+ */
+import java.nio.charset.StandardCharsets;
```

Comment:
A blank line between 191 and 192 would be useful here and between 216 and 217, and generally between the end of each method and the start of the comment for the next method or set of definitions.

Also would it be possible to add the server scenario letter A-F into the big comment table that describes each client and server scenario? That would help understanding each test scenario by looking in one place.